### ISSUE TITLE
Nuget package?

### ISSUE THREAD

**Issue by DenisBalan** [Tolyandre/serilog-throw-context-enricher#3]

**Comment by Tolyandre** [comment]
Hi,
Here are the nuget package https://www.nuget.org/packages/Serilog.ThrowContext. There is a link in the title row

**Comment by morgrowe** [comment]
> Hi, Here are the nuget package https://www.nuget.org/packages/Serilog.ThrowContext. There is a link in the title row

It's quite difficult to find as the convention seems to be `Serilog.Enrichers.<EnricherName>` rather than `Serilog.<EnricherName>`. 

This package is amazing, though. We've been battling with this problem alllllll day. It was only by chance I discovered this. Thank you for this. :)

**Comment by Tolyandre** [comment]
Yes, I agree chosen name is not that good. Probably I can deprecate this package on nuget and reupload it under name `Serilog.Enrichers.ThrowContext`. But I have doubts:
- I don't know how deprecation works. It would be confusing to find 2 identical packages on nuget.
- Not sure I can upload under Serilog prefix nowdays, as nuget may reserve prefixes now. Probably I should rename to `<MyName>.Serilog.Enrichers.<EnricherName>` or give this repo and package to real Serilog contributors.
